### ISSUE TITLE
lora训练的loss无法下降，一直维持在1.0左右

### ISSUE THREAD

**Issue by wqn1** [LianjiaTech/BELLE#405]
训练脚本如下，目前任务能正常跑起来，但是loss一直无法下降，大概一天loss只从1.3降到了1.0，超参学习率调整过，lora-alpha也调过，一直没有合理的效果，想问下你们之前调试的时候有没有遇到过类似的问题，训练设备时2块A100 80G的显卡：
OUTPUT_PATH=checkpoint/20230524
ZERO_STAGE=2
export CUDA_VISIBLE_DEVICES="0,1"
echo $OUTPUT_PATH
echo $ZERO_STAGE
mkdir -p $OUTPUT_PATH

model_name_or_path=
lora_module_name="q_proj,k_proj,v_proj,o_proj,down_proj,gate_proj,up_proj"

echo ${lora_module_name}

nohup deepspeed /BELLE/train/dschat_train_v1/main.py \
   --sft_only_data_path data.json\
   --eval_data_file llm_identity.json \
   --data_split 10,0,0 \
   --model_name_or_path ${model_name_or_path} \
   --per_device_train_batch_size 4 \
   --per_device_eval_batch_size 1 \
   --max_seq_len 1024 \
   --learning_rate 3e-5 \
   --weight_decay 0. \
   --num_train_epochs 3 \
   --show_loss_step 128 \
   --gradient_accumulation_steps 4 \
   --lr_scheduler_type cosine \
   --num_warmup_steps 100 \
   --seed 1234 \
   --gradient_checkpointing \
   --zero_stage $ZERO_STAGE \
   --lora_dim 16 \
   --lora_alpha 16 \
   --lora_droppout 0.05 \
   --lora_module_name ${lora_module_name} \
   --deepspeed \
   --output_dir ${OUTPUT_PATH} \
   > $OUTPUT_PATH/training230516.log 2>&1 &

**Comment by ruleGreen** [comment]
有，很奇怪
